packages/app-store/qr_code/components/EventTypeAppCardInterface.tsx

```diff
@@ -43,24 +18,13 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
       }}
       switchChecked={enabled}
       teamId={eventType.team?.id || undefined}>
-      <div className="flex w-full flex-col gap-5 text-sm">
-        <div className="flex w-full">
-          <TextField
-            name="hello"
-            disabled={disabled}
-            value={additionalParameters}
-            onChange={(e) => setAdditionalParameters(e.target.value)}
-            label={t("additional_url_parameters")}
-            containerClassName="w-full"
-          />
-        </div>
-
-        <div className="max-w-60 flex items-baseline gap-2">
-          <QRCode size={256} data={eventTypeURL} />
-          <QRCode size={128} data={eventTypeURL} />
-          <QRCode size={64} data={eventTypeURL} />
-        </div>
-      </div>
+      <EventTypeAppSettingsInterface
+        eventType={eventType}
+        slug={app.slug}
+        disabled={disabled}
+        getAppData={getAppData}
+        setAppData={setAppData}
+      />
```

Comment:
need to split `packages/app-store/{app_dirname}/components/EventTypeAppCardInterface.tsx` and add EventTypeAppSettingsInterface for all the apps